aiormq/channel.py

```diff
@@ -609,6 +609,10 @@ async def basic_publish(
         timeout: TimeoutType = None,
         wait: bool = True,
     ) -> Optional[ConfirmationFrameType]:
+
+        if len(routing_key) > 255:
```

Comment:
Have to add similar checks for `exchange_bind`, `exchange_unbind`, `queue_bind` and `queue_unbind` I guess.

Might be reasonable write the function `_check_routing_key` and call it for this cases.

Reply:
Good point. Done.